packages/frontend/src/components/ControlledAutocomplete/index.tsx

```diff
@@ -145,6 +146,22 @@ function ControlledAutocomplete(
                 freeSolo={freeSolo}
                 options={options}
                 value={getOption(options, field.value, freeSolo)}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    InputProps={{
+                      ...params.InputProps,
+                      endAdornment: (
+                        <>
+                          {loading && (
+                            <Spinner fontSize={24} color="primary.600" />
```

Comment:
nit: `spinner={<Spinner fontSize={24} color="primary.600" />}` can also add this to the button above for refresh items so that the spinner used is consistent?

Reply:
good thinking!